### PR TITLE
<SidebarSectionItem/><SidebarSectionTitle/><SidebarDivider/> - sidebar experiment for reduce horizontal padding and height for sidebar items

### PR DESCRIPTION
**Pull request by katyamois**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [x] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_7025/

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_7025

File: packages/wix-style-react/src/SidebarSectionItem/SidebarSectionItem.st.css [hunk 1]
```diff
@@ -141,6 +141,19 @@
   margin: 9px 0 9px 0;
 }
 
+.root:useBmSidebarNewDesign {
```
Comment (miby-wix) [inline packages/wix-style-react/src/SidebarSectionItem/SidebarSectionItem.st.css:144]: where is it wrapped on SidebarSecrtionItem with the provider? was it on previous PR?

Comment (miby-wix) [reply packages/wix-style-react/src/SidebarSectionItem/SidebarSectionItem.st.css:144]: Pending on this one only.

**Review by miby-wix — CHANGES_REQUESTED** [review]

File: packages/wix-style-react/src/SidebarSectionItem/SidebarSectionItem.st.css (hunk 1 above)
Comment (katyamois) [reply packages/wix-style-react/src/SidebarSectionItem/SidebarSectionItem.st.css:144]: yes it was on previous pr

**Review by miby-wix — APPROVED** [review]

**Review by mykas — APPROVED** [review]

**Pull request merged**